golang/lambda/example_lambda_configurator.go

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+package lambda
+
+import (
+	"encoding/json"
+	"github.com/kurtosis-tech/kurtosis-lambda-api-lib/golang/lib/lambda"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+)
+
+type ExampleLambdaConfigurator struct{}
+
+func NewExampleLambdaConfigurator() *ExampleLambdaConfigurator {
+	return &ExampleLambdaConfigurator{}
+}
+
+func (t ExampleLambdaConfigurator) ParseParamsAndCreateLambda(serializedCustomParamsStr string) (lambda.Lambda, error) {
+	serializedCustomParamsBytes := []byte(serializedCustomParamsStr)
+	var args ExampleLambdaArgs
+	if err := json.Unmarshal(serializedCustomParamsBytes, &args); err != nil {
+		return nil, stacktrace.Propagate(err, "An error occurred deserializing the Lambda serialized custom params with value '%v", serializedCustomParamsStr)
+	}
+
+	//TODO VALIDATE ARGUMENTS log level can't be an empty string
+
+	lambda := NewExampleLambda()
+
+	err := setLogLevel(args.LogLevel)
```

Comment:
I'd also recommend pushing the loglevel-setting as close after arg-parsing as possible - that way, there's less chance of someone accidentally inserting a log message in between and being confused why it doesn't get outputted

Reply:
ok, thanks